src/chainparams.cpp

```diff
@@ -443,6 +442,11 @@ class CTestNetParams : public CChainParams {
         nDefaultPort = 58811;
         nPruneAfterHeight = 1000;
 
+        /** Timestamp when to switch to ProgPow, RandomX, Sha256D. UTC based **/
+        // TODO update when ready for testnet launch
+        nPowUpdateTimestamp = 8578938400; // Monday, January 13, 2020 6:00:00 PM
+        nPowTimeStampActive = 8578938400; // Used by block.h for serialization
```

Comment:
These need to be reduced to 4294967295 so that we don't actually inadvertently turn it on due to the value exceeding a 32 bit boundary.

Reply:
Done